### DISCUSSION TITLE
ctrl+z is not working in the body field if you send one request

### DISCUSSION THREAD

**Discussion by ademcayir** [Kong/insomnia#7562] [Ideas] [answered]
When you are playing with the body input, ctrl+z is working until you send one request. After you send request once, ctrl+z is not working anymore. But, usually you need to check old and new version of the input.

use case:
- open a method
- go body part (I use JSON if it matters)
- edit the input, try ctrl+z. it is working
- then press "send" button
- go back body part. ctrl+z doesnt remember anymore your previous request

**Comment by subnetmarco** [comment, marked as answer]
This has been fixed in 9.3

**Reply by DanGM96** [reply]
This exact problem is present on `11.6.1` (Windows), I just downloaded Insomnia so I'm not sure which version reintroduced this.

**Reply by UberVeng** [reply]
Same problem on windows 11.6.1

**Reply by rinaldilucas** [reply]
Still happens on the 12.0.0

**Reply by israelbarcelos-sebratel** [reply]
still happens on the 12.4.0
